src/client/content_cao.cpp

```diff
@@ -924,7 +924,9 @@ void GenericCAO::updateNodePos()
 void GenericCAO::step(float dtime, ClientEnvironment *env)
 {
 	if (m_animated_meshnode) {
-		m_animated_meshnode->animateJoints();
+		if (!m_stop_animations) {
```

Comment:
This makes it an object-wide setting, even though the lua code is generally a per-bone operation. Setting it on mutiple bones with conflicting parameters results in confusing behavior.

Reply:
Also, does it restore the bind pose correctly after a mesh has already been animated? If not, it would be dependent on when the animation was frozen, and that's also pretty useless.